db/migrate/20210414180753_change_url_to_json.rb

```diff
@@ -0,0 +1,10 @@
+class ChangeUrlToJson < ActiveRecord::Migration[6.0]
+  def up
+    remove_column :tasks, :url
```

Comment:
This kind of migration can't get to production server, as we will lose all data in that column